src/main/java/org/broadinstitute/consent/http/resources/VoteResource.java

```diff
@@ -87,10 +83,56 @@ public Response updateVotes(
                 return createExceptionResponse(new NotFoundException());
             }
 
-            List<Vote> updatedVotes = voteService.updateVotesWithValue(votes, vote, rationale);
+            List<Vote> updatedVotes = voteService.updateVotesWithValue(votes, voteUpdateInfo.getVote(), voteUpdateInfo.getRationale());
             return Response.ok().entity(updatedVotes).build();
         } catch (Exception e) {
             return createExceptionResponse(e);
         }
     }
+
+    static class VoteUpdateInfo {
```

Comment:
Thinking out loud here. We have this simple model class, and I have one for a similar method in my PR. It might make sense to pull these out into a separate model package specific to the resources directly. Another possibility would be in `org/broadinstitute/consent/http/models/vote`